src/shared/ui/index.ts

```diff
@@ -0,0 +1,18 @@
+//assets
+export { Logo } from "./assets/Logo";
+export { Google } from "./assets/Google";
+export { DeviceMobile } from "./assets/DeviceMobile";
+export { PhoneNumberAuthIllustration} from "./assets/PhoneNumberAuthIllustration";
+export {DownArrow} from "./assets/DownArrow";
+export {CloseIcon} from "./assets/CloseIcon";
+export {VerificationImg} from "./assets/VerificationImg";
+
+//consts
+export {StyleGuide} from "./stylesConsts/stylesConsts";
+
+//components
+export {CustomInput} from "./Components/CustomInput/ui/CustomInput";
+export {CustomButton} from "./Components/CustomButton/ui/CustomButton";
+export { Map } from "./Components/Map/ui/Map";
+export {CustomImage} from "./Components/CustomImage/ui/CustomImage";
+export {CustomCheckBox} from "./Components/CustomCheckBox/ui/CustomCheckBox";
```

Comment:
remove comments

Reply:
i think that comments here are acceptable for better structuring